source/_posts/2015-03-26-creating-a-dynamic-single-page-app-for-our-genome-team-using-react.markdown

```diff
@@ -0,0 +1,294 @@
+---
+layout: post
+title: "Trying out React"
+date: 2015-03-26 10:42
+comments: true
+categories: [React, JavaScript, CoffeeScript]
+author: Sarah Weir
+github-url: https://www.github.com/sweir27
+twitter-url: http://twitter.com/sweir27
+---
+We recently picked up a Rails application that was a few features away from completion. This application allows our [Genome Team](https://www.artsy.net/about/the-art-genome-project) to classify multiple artworks based on visual and art historical characteristics. These characteristics, or "genes", can be added, removed, and changed for any of the artworks on the panel. 
+
+![screenshot](http://cl.ly/image/310O3U3J3T2c)
+
+Our genomers are masters of efficiency, and over the years they have worked closely with our engineering team to tailor a dynamic interface to their specific workflow. 
```

Comment:
Minor: you start by saying "we recently picked up a Rails app...", assuming we = engineering team, but then you say "with our engineering team", I guess that should just say with "us", or something like that.